src/git.rs

```diff
@@ -194,14 +194,17 @@ pub fn checkout_branch(
                 ));
             };
             debug!("Checking out branch: {}", local_branch_name);
+            let mut checkout_args = vec!["checkout"];
             let origin_with_remote = format!("{}/{}", remote_name, remote_branch_name);
-            let remote_ref = if is_virtual_remote_branch {
-                &local_branch_name
+            if is_virtual_remote_branch {
+                checkout_args.push(&local_branch_name)
             } else {
-                &origin_with_remote
+                checkout_args.push("-b");
+                checkout_args.push(&local_branch_name);
+                checkout_args.push(&origin_with_remote);
             };
-            trace!("Checking '{}' as '{}'", remote_ref, local_branch_name);
-            match cmd!("git", "checkout", "-b", &local_branch_name, &remote_ref).run() {
+            trace!("Checking out branch: {}", local_branch_name);
```

Comment:
For future debugging, can you move this trace call into each of the above branches, adjusting the format string to reflect the different locals?

Reply:
Done.